CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
```

Comment:
Line 16/16 Don't forget to add the NeMo-Aligner "Next Version"

Line 19/19 Use full product name.

Added support for Megatron Core's distributed optimizer, which can be configured using `++model.optim.name=mcore_distributed_optim`.

Reply:
> Don't forget to add the NeMo-Aligner "Next Version"

Thanks for the reminder. We will do this before the next release.